test/testsNumber.test.js

```diff
@@ -0,0 +1,18 @@
+import numberGame from '../src/numberGame.js';
+
+const test = QUnit.test;
+
+test('compair numbers', function(assert) {
+    //Arrange
+    // Set up your parameters and expectations
+    const guess = 6;
+    const correct = 6;
+    const expected = 0;
```

Comment:
make sure to test for the other cases that your test number function handles, such as if the number is too low or too high